ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/MetacardApplication.java

```diff
@@ -1057,7 +1057,8 @@ private void attemptDeleteDeletedMetacard(String id) {
     LOGGER.trace("Attemping to delete metacard [{}]", id);
     Filter tags =
         filterBuilder.attribute(Metacard.TAGS).is().like().text(DeletedMetacard.DELETED_TAG);
```

Comment:
Not related to this PR, but I see several instances of using like filters on tags in this repo. I have a feeling this wasn't  thought through and probably should be changed to an equals filter.  Thoughts?